### ISSUE TITLE
Error: Cannot find module 'pouchdb-admins' /w Node 4.3.0

### ISSUE THREAD

**Issue by schultyy** [hoodiehq/hoodie#491]
I start a new project with:

```
npm init -y
```

I install hoodie:

```
$ npm install hoodie --save
```

then my `package.json` looks like:

``` json
{
  "name": "hoodie-test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "hoodie",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "hoodie": "^22.0.3"
  }
}
```

I install all dependencies with `npm install`. Then I try to start the application with:

```
$ npm start
```

This results in the following error:

```
> hoodie-test@1.0.0 start /Users/janschulte/temp/hoodie-test
> hoodie

module.js:328
    throw err;
    ^

Error: Cannot find module 'pouchdb-admins'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/janschulte/temp/hoodie-test/node_modules/hoodie/server/config/db/pouchdb.js:8:14)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/janschulte/.avm/node/4.3.0/bin/node" "/Users/janschulte/.avm/node/bin/npm" "start"
npm ERR! node v4.3.0
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE
npm ERR! hoodie-test@1.0.0 start: `hoodie`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hoodie-test@1.0.0 start script 'hoodie'.
npm ERR! This is most likely a problem with the hoodie-test package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     hoodie
npm ERR! You can get their info via:
npm ERR!     npm owner ls hoodie-test
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/janschulte/temp/hoodie-test/npm-debug.log
```

When I do `npm install pouchdb-admins --save` everything works fine.

**Comment by schultyy** [comment]
Forgot to add: I run Mac OS X 10.10.5.

**Comment by gr2m** [comment]
Thanks @schultyy I’m looking into it right now

**Comment by gr2m** [comment]
I could reproduce the error with node@4.3.0 and npm@2.14.12, looking into where the problem is

**Comment by gr2m** [comment]
okay found and fixed the issue via https://github.com/hoodiehq/hoodie/pull/492. Released as [hoodie@22.0.4](https://github.com/hoodiehq/hoodie/releases/tag/v22.0.4)
